packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -1257,3 +1340,43 @@ const getLegendProps = (data: Data[], layout: Partial<Layout> | undefined) => {
       layout?.showlegend === false || (layout?.showlegend !== true && legends.length < 2) ? true : hideLegends,
   };
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isInvalidValue = (value: any) => {
```

Comment:
could you add the corresponding examples in test app and share the playwright report

Reply:
Added the test examples here: [Add examples with some invalid data points by krkshitij · Pull Request #159 · microsoft/fluentui-charting-contrib](https://github.com/microsoft/fluentui-charting-contrib/pull/159)